ODIN_II/SRC/verilog_flex.l

```diff
@@ -216,6 +216,7 @@ vPUNCT 	[\?\:\|\^\&\<\>\-\*\/\%\(\)\{\}\[\]\~\!\;\#\,\.\@\=\+]
 <INITIAL>"xnor"						{ MP(); return vXNOR;}
 <INITIAL>"xor"						{ MP(); return vXOR;}
 <INITIAL>"specparam"				{ MP(); return vSPECPARAM;}
+<INITIAL>"buf"				        { MP(); return vBUF;}
```

Comment:
I just noticed that line 236 still contains "buf" also and gives off a warning. can you make another PR for this. Thanks